### PR TITLE
Feat: change private methods to admin methods

### PR DESCRIPTION
**Pull request by hskang9**
## Description

This PR changes the following 3 Aurora Engine's private methods to admin methods for Aurora Silo and DAO migration:

- new_eth_connector
- set_eth_connector_contract_data
- set_paused_flags

Changing these methods required to make an account to deploy then set owner to be the account to deploy like the updated deployment [code](https://github.com/hskang9/aurora-silo-tests/blob/9b3faa82dae0a865095ff4cb57cf41dd6f97185b/src/common.rs#L42). Adding `set_owner` method in the engine is expected after this PR.

## Performance / NEAR gas cost considerations

Performance has not been considered as this is just an access control change. Gas monitoring would be appreciated.

## Testing

NEAR Workspace test on testing the methods can be found [here](https://github.com/hskang9/aurora-silo-tests).

## How should this be reviewed

Verify compiled engine code in the workspace test repo, Check code in the workspace test, run tests in the repo, and consider gas cost optimization on code changes.

## Additional information

This is a part of Aurora DAO migration and Silo milestone.

### CODE REVIEW

**Comment by joshuajbouw** [comment]
Fix tests please.

File: engine/src/lib.rs [hunk 1]
```diff
@@ -846,7 +855,12 @@ mod contract {
     #[no_mangle]
     pub extern "C" fn set_paused_flags() {
         let io = Runtime;
-        io.assert_private_call().sdk_unwrap();
+        // Only the owner can initialize the EthConnector
```
Comment (birchmd) [inline engine/src/lib.rs:858]: Wrong comment

**Review by birchmd — APPROVED** [review]

**Review by aleksuss — APPROVED** [review]
LGTM, but change the comment in the `set_paused_flags` function.

**Comment by hskang9** [comment]
@aleksuss @birchmd comment is removed.

**Review by aleksuss — APPROVED** [review]

**Comment by hskang9** [comment]
@joshuajbouw is admin change delayed as well?

**Comment by hskang9** [comment]
@joshuajbouw the code is now ready to be merged.

**Pull request merged**